pom.xml

```diff
@@ -4967,6 +4977,10 @@
                 <artifactId>wss4j-ws-security-common</artifactId>
                 <version>${version.org.apache.ws.security}</version>
                 <exclusions>
+                    <exclusion>
+                        <groupId>org.apache.geronimo.javamail</groupId>
+                        <artifactId>geronimo-javamail_1.4_mail</artifactId>
+                    </exclusion>
```

Comment:
We might want to verify the one below this is valid. It's possible the `groupId` below was just a mistype.

Reply:
verified, dunno if mistyped or simply outdated but no such dep exists there